src/utils/db/lock-and-query-payments/index.ts

```diff
@@ -23,94 +24,99 @@ export async function lockAndQueryPayments({
             },
-            orderBy: {
-              createdAt: 'asc',
-            },
+            AdminWallets: true,
+            FeeReceiverNetworkWallet: true,
+            PaymentSourceConfig: true,
+          },
+          orderBy: {
+            createdAt: 'asc',
           },
-          AdminWallets: true,
-          FeeReceiverNetworkWallet: true,
-          PaymentSourceConfig: true,
-        },
-        orderBy: {
-          createdAt: 'asc',
-        },
-      });
-      const sellingWallets: HotWallet[] = [];
+        });
+        const sellingWallets: HotWallet[] = [];
 
-      const newPaymentSources = [];
-      for (const paymentSource of paymentSources) {
-        const paymentRequests = [];
-        const minCooldownTime = paymentSource.cooldownTime;
-        for (const paymentRequest of paymentSource.PaymentRequests) {
-          if (
-            paymentRequest.sellerCoolDownTime >
-            Date.now() - minCooldownTime
-          ) {
-            continue;
-          }
+        const newPaymentSources = [];
+        for (const paymentSource of paymentSources) {
+          const paymentRequests = [];
+          const minCooldownTime = paymentSource.cooldownTime;
+          for (const paymentRequest of paymentSource.PaymentRequests) {
+            if (
+              paymentRequest.sellerCoolDownTime >
+              Date.now() - minCooldownTime
+            ) {
+              continue;
+            }
 
-          const wallet = paymentRequest.SmartContractWallet;
-          if (
-            wallet != null &&
-            !sellingWallets.some((w) => w.id === wallet.id)
-          ) {
-            const result = await prisma.hotWallet.update({
-              where: { id: wallet.id, deletedAt: null },
-              data: { lockedAt: new Date() },
+            const wallet = paymentRequest.SmartContractWallet;
+            if (
+              wallet != null &&
+              !sellingWallets.some((w) => w.id === wallet.id)
+            ) {
+              const result = await prisma.hotWallet.update({
+                where: { id: wallet.id, deletedAt: null },
+                data: { lockedAt: new Date() },
+              });
+              wallet.pendingTransactionId = result.pendingTransactionId;
+              sellingWallets.push(wallet);
+              paymentRequests.push(paymentRequest);
+            }
+          }
+          if (paymentRequests.length > 0) {
+            newPaymentSources.push({
+              ...paymentSource,
+              PaymentRequests: paymentRequests,
             });
-            wallet.pendingTransactionId = result.pendingTransactionId;
-            sellingWallets.push(wallet);
-            paymentRequests.push(paymentRequest);
           }
         }
-        if (paymentRequests.length > 0) {
-          newPaymentSources.push({
-            ...paymentSource,
-            PaymentRequests: paymentRequests,
-          });
-        }
+        return newPaymentSources;
+      } catch (error) {
```

Comment:
I am not sure this is necesarrily useful, as it is handled outside of here. Also try/catch/ within the tx does not nexessarily make sense, if needed, then probably around here